website/project/decorators.py

```diff
@@ -329,3 +330,15 @@ def wrapped(*args, **kwargs):
 
     # Return decorator
     return wrapper
+
+
+def dev_only(func):
```

Comment:
Restricting certain views to dev-only was done at the request of @brianjgeiger ; pinging him for followup.